src/write.c

```diff
@@ -811,6 +828,21 @@ static avifImage * avifImageCopyAndPad(const avifImage * srcImage, uint32_t dstW
     return dstImage;
 }
 
+static int avifQualityToQuantizer(int quality, int minQuantizer, int maxQuantizer)
+{
+    int quantizer;
+    if (quality == AVIF_QUALITY_DEFAULT) {
+        // In older libavif releases, avifEncoder didn't have the quality and qualityAlpha fields.
+        // Supply a default value for quantizer.
+        quantizer = (minQuantizer + maxQuantizer) / 2;
+        quantizer = AVIF_CLAMP(quantizer, 0, 63);
```

Comment:
This is a tough call. The `AVIF_CLAMP` calls are easy to deal with. For example,

```
        quantizer = AVIF_CLAMP(quantizer, AVIF_QUANTIZER_BEST_QUALITY, AVIF_QUANTIZER_WORST_QUALITY);
```

The difficulty is in the conversion formula in line 841:

```
        quantizer = ((100 - quality) * 63 + 50) / 100;
```

The 63 in the formula is actually `(AVIF_QUANTIZER_WORST_QUALITY - AVIF_QUANTIZER_BEST_QUALITY)`.

And we would need to add a `+ AVIF_QUANTIZER_BEST_QUALITY` term (which expands to + 0 and can be omitted).

So it would look like:

```
        quantizer = ((100 - quality) * (AVIF_QUANTIZER_WORST_QUALITY - AVIF_QUANTIZER_BEST_QUALITY) + 50) / 100
                    + AVIF_QUANTIZER_BEST_QUALITY;
```

And we would also need to deal with the 100 and 50 in the formula.